### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by unasuke**
Add new icons.

- tmix-scaling-down
- tmix-design-finish

### CODE REVIEW

File: app/assets/stylesheets/tmix-icon.css [hunk 1]
```diff
@@ -23,6 +23,12 @@ i {
   -moz-osx-font-smoothing: grayscale;
 }
 
+.tmix-scaling-down:before {
+  content: "\e96d";
+}
+.tmix-design-finish:before {
+  content: "\e96e";
+}
```
Comment (hshimoyama) [inline app/assets/stylesheets/tmix-icon.css:31]: ここの順番てどういう並びになってるんだろ？ content の並びと違うのね

**Review by hshimoyama — APPROVED** [review]
🙆‍♂️

File: app/assets/stylesheets/tmix-icon.css (hunk 1 above)
Comment (unasuke) [reply app/assets/stylesheets/tmix-icon.css:31]: Yes. Order is conform by icomoon's generated CSS.

I guess class order is by descending with update.

**Comment by unasuke** [comment]
Thank you!

**Pull request merged**